clang/lib/Sema/Sema.cpp

```diff
@@ -267,6 +267,12 @@ void Sema::Initialize() {
   if (getLangOpts().SYCLIsDevice) {
     addImplicitTypedef("__ocl_event_t", Context.OCLEventTy);
     addImplicitTypedef("__ocl_sampler_t", Context.OCLSamplerTy);
+#define stringization(s) #s
```

Comment:
a) Please use capital letter for the macro. For example STRINGIZE or STRINGIZATION

b) The name 'stringization' ,might be already defined by the code above of include files. You can reduce that chance by adding SEMA suffix and also have a check like this:
`#ifdef SEMA_STRINGIZE
#error "Undefine SEMA_STRINGIZE macro."
#endif
#define SEMA_STRINGIZE(s) #s
`

Reply:
done.